free5gc-operator/operator/role-leader-election.yaml

```diff
@@ -1,11 +1,16 @@
 apiVersion: rbac.authorization.k8s.io/v1
 kind: Role
+
 metadata:
+  namespace: free5gc
+  name: free5gc-operator-leader-election-role
   labels:
-    app.kubernetes.io/name: free5gc-operator
+    app.kubernetes.io/name: role
+    app.kubernetes.io/instance: free5gc-operator-leader-election-role
```

Comment:
Instance here should be the instance of the application. So, probably if we want to set this properly we should have a kpt set-labels that sets it to the package name.

Reply:
I was aiming for consistency with what we have in the free5gc repo for now. Generally we want consistent labels across all Nephio, not sure if have that now...

Reply:
Definitely not. But these are standard K8s labels so we should either not use them, or use them as expected. External K8s tooling will expect certain semantics from these labels. For example, `kubectl cluster-info` I believe uses these labels.